### PR TITLE
chore(maintain): Integrate Coveralls

### PR DESCRIPTION
**Pull request by ajaps**
**What does this PR do?**
- This PR displays  coverage badge into the Readme file of the project

**Description of Task to be completed?**
- Integrates coveralls, gets test result from travis and displayed badges on readme

 **How should this be manually tested?**
- npm run coveralls

**Any background context you want to provide?**
Travis CI finally builds without errors and the badge is displayed on the readme file.  The link to the badge in the readme file was changed from pointing to my branch name to the staging branch.
Therefore the badge wouldn't display on 'Chore/146622687/coveralls-build-displays-readme' branch because the badge is pointing to the 'staging' branch which has errors on travis build - this PR would fix it.

**What are the relevant pivotal tracker stories?**
- #146622687

#### Questions:
- None

### CODE REVIEW

File: test/user/model.js [hunk 1]
```diff
@@ -24,12 +24,12 @@ describe('<Unit Test>', function() {
         });
 
         describe('Method Save', function() {
```
Comment (seyi-adeleke) [inline test/user/model.js:26]: I don't fully understand why this test was commented out. It tests an essential feature so I believe your proposed changes should work with the test before merging to ensure we can run the test successfully in the future.

File: test/user/model.js [hunk 2]
```diff
@@ -24,12 +24,12 @@ describe('<Unit Test>', function() {
...
-            it('should be able to save whithout problems', function(done) {
```
Comment (seyi-adeleke) [inline test/user/model.js]: I don't fully understand why this test was commented out. It tests an essential feature so I believe your proposed changes should work with the test before merging to ensure we can run the test successfully in the future.

File: test/user/model.js (hunk 1 above)
Comment (ajaps) [reply test/user/model.js:26]: Itua who worked on integrating the test into Travis, confirmed the said test was failing due to the inability to connect to the database.
He requested the test be disabled for now - hence the commenting of the test case

Comment (fokosun) [reply test/user/model.js:26]: @seyi-adeleke This make sense. Kindly approve the PR so we can get this sorted and out of the way. We'll create another story to fix the test.

**Review by seyi-adeleke — APPROVED** [review]

**Pull request merged**